src/controls/dashboard/widget/IWidget.ts

```diff
@@ -96,5 +96,20 @@ export interface IWidgetBodyContent {
  * Widget link
  */
 export interface IWidgetLink {
+  /**
+   * Link href
+   */
   href: string;
+  /**
+   * The text to display for the link, if not provided, the default text will be used
+   */
+  title?: string;
+  /**
+   * The color of the link, if not provided, the default color will be used
```

Comment:
I made some tests regarding the `color` prop of Fluent UI React Northstar `Text` component... Looks like available colors are limited to what's displayed in the definition / dropdown prop on the [doc page](https://fluentsite.z22.web.core.windows.net/0.66.2/components/text/definition).

More info [here](https://github.com/microsoft/fluentui/blob/master/packages/fluentui/react-northstar/src/themes/teams/colors.ts#L534).

I'd suggest to update documentation to indicate that available colors can be found on the official doc of the `Text` control, in order to avoid wrong feedback. Moreover, `default` color is not what's currently selected (as there's actually an existing `default` value in the list). Maybe indicate the `brand` one instead or update the code in `WidgetFooter` to set `default` in color prop?

Reply:
Updated in the next commit.